### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by Haywhizzz**
In this PR I have:

- Implemented the app component

- Implemented the Display component

- Implemented the Button component

- Implemented the ButtonPanel component

- I have non-functional buttons in my app browser at the moment

### CODE REVIEW

File: src/components/Display.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  const { result } = props;
+  return (
+    <div>{result}</div>
+  );
+}
+
+Display.propTypes = {
+  result: PropTypes.string.isRequired,
+};
+
```
Comment (addod19) [inline src/components/Display.js:17]: Kindly add/set the default value of the result props to `0`

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @Haywhizzz, great job on this project :+1: The PR looks descriptive :100: 
The App, Button, and ButtonPanel component have been implemented successfully :1st_place_medal: However, there are a few comments that needs to be addressed, kindly check the comments below:

- Kindly use the right linter configuration file. Here is the right [file](https://github.com/microverseinc/linters-config/blob/master/react-redux/.github/workflows/linters.yml)
- The README should be improved to have a professional feel, other than having the default template. The README should include information about how to clone the project :+1: 


Once these are addressed, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: src/components/Display.js (hunk 1 above)
Comment (Haywhizzz) [reply src/components/Display.js:17]: I have set the default value of the result props to `0` and also implemented the changes suggested

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**